Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+gem 'pry'
```

Comment:
We probably don't need this in this PR.

Reply:
Isn't pry already part of MSF? Or is that just a dev group thing?

Reply:
I worked myself into a detached head state as far as git is concerned.  Had never seen that and googled how to get the work I'd done back into the proper branch.  The fix action snuck this little guy in.  I never explicitly `git add`ed the Gemfile, however it got included and I didn't realize.  I'll get it out of the PR shortly.  Sorry about that.

Reply:
Removed the modified Gemfile.